rand_pcg/src/pcg128.rs

```diff
@@ -40,6 +40,35 @@ pub struct Lcg128Xsl64 {
 pub type Pcg64 = Lcg128Xsl64;
 
 impl Lcg128Xsl64 {
+    /// Multi-step advance functions (jump-ahead, jump-back)
```

Comment:
We should probably mention that this is equivalent to `delta` times calling `next_u64`.